### DISCUSSION TITLE
Attributter - Helsepersonellets strukturelle rolle - structural_role

### DISCUSSION THREAD

**Discussion by eirikbroen** [NorskHelsenett/Tillitsrammeverk#9] [General]
Antar at de fleste EPJ systemer ikke har en mapping fra brukerrolle til de tre foreslåtte verdiene. I og med at dette attributtet er satt opp som obligatorisk bør vi forsøke å gjøre innføring så enkel som mulig. Er tanken her at denne verdien angir hvilken rolle personell har i i en gitt behandlingsrelasjon og at verdi ikke kan utledes fra HPR informasjon eller mangel på denne?

**Comment by OK-SverreM** [comment]
For første versjon av tillitsrammeverket - til bruk i 2023, og det som skal implementeres som del av behovskøelement # 27 - så vil jeg foreslå at vi legger dette attributtet til side, og at det i hvert fall **ikke** kan være obligatorisk.
Som Eirik er inne på, er det til dels overlapp / avhengigheter mellom "structural_role" og informasjonen om autorisasjoner/lisenser. 
Oppslag i HPR vil kunne gi svar på om helsepersonellet er autorisert / har lisens som helsepersonell. Hvilken merverdi gir da attributtet "structural_role", og i hvilke scenarioer er det viktig å oppgi dette (eksempler)?

**Comment by steinarnoem** [comment]
I arbeidsmøte 02.03.2023 ble det besluttet at informasjonselementet som beskriver helsepersonellets "strukturelle rolle" tas ut av spesifikasjonen. https://github.com/NorskHelsenett/Tillitsrammeverk/blob/main/meeting_minutes/2023-03-02%20informasjonsmodell.md.

Diskusjonstråden låses, og det er opprettet et issue basert på konklusjonen.
